types/amplitude-js/index.d.ts

```diff
@@ -35,6 +35,7 @@ export interface Config {
     saveParamsReferrerOncePerSession?: boolean;
     secureCookie?: boolean;
     sessionTimeout?: number;
+    sameSiteCookie?: string;
```

Comment:
according to MDN sameSite can have three valueshttps://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite
```suggestion
    sameSiteCookie?: 'Lax' | 'Strict' | 'None';
```
according to MDN https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite